app/services/pledge_form_generator.rb

```diff
@@ -1,12 +1,13 @@
 class PledgeFormGenerator
   include ActionView::Helpers::NumberHelper # for currency formatting
 
-  attr_reader :patient, :user, :case_manager_name
+  attr_reader :patient, :user, :case_manager_name, :fund
```

Comment:
Adjusting this is one of the few heavier things here. Test suite should keep us mostly safe but want to generate a pledge tobe sure.

Reply:
Generated a pledge, confirmed it worked!